dpctl/program/_program.pxd

```diff
@@ -33,26 +33,26 @@ cdef class SyclKernel:
         kernel.
     '''
     cdef DPCTLSyclKernelRef _kernel_ref
-    cdef const char *_function_name
+    cdef str _function_name
     cdef DPCTLSyclKernelRef get_kernel_ref (self)
 
     @staticmethod
-    cdef SyclKernel _create (DPCTLSyclKernelRef kref)
+    cdef SyclKernel _create (DPCTLSyclKernelRef kref, str name)
 
 
 cdef class SyclProgram:
-    ''' Wraps a sycl::program object created from an OpenCL interoperability
-        program.
+    ''' Wraps a sycl::kernel_bundle<sycl::bundle_state::executable> object created from
```

Comment:
Something is off with grammar "created from using"

Reply:
Yes, should be "created by using".

Reply:
Pushed the change. Resolving.